Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdater.cs

```diff
@@ -57,31 +57,32 @@ public void Update(CancellationToken cancellationToken)
                 {
                     DebugLogger.LogWarningFormat("Strategy caused exception, being handled by fallback: {0}", ex.Message);
 
-                    HandleFallback(cancellationToken);
+                    if (!TryHandleFallback(cancellationToken))
+                    {
+                        throw;
+                    }
                 }
             }
         }
 
-        private void HandleFallback(CancellationToken cancellationToken)
+        private bool TryHandleFallback(CancellationToken cancellationToken)
         {
             StrategyType fallbackType = _strategyResolver.GetFallbackStrategy(_strategy.GetStrategyType());
-
-            if (fallbackType == StrategyType.None)
+            bool HasFallback = fallbackType != StrategyType.None;
```

Comment:
Local variables should use lowerCamelCase. But beside that, everything is OK!

Good job @sp0lsh, thank you for adding this feature :-)